torch/_functorch/aot_autograd.py

```diff
@@ -2056,6 +2114,25 @@ def compute_overlapping_inputs(fwd_inputs, aliased_input_indices):
                 actual_aliased_indices.add(j_)
     return actual_aliased_indices
 
+
+def _check_if_mutation_can_be_handled_in_fw_graph(input_info: InputAliasInfo):
+    return input_info.mutates_data and not input_info.mutates_metadata and not input_info.requires_grad
+
+
+def _check_if_mutation_will_be_returned_in_runtime(input_info: InputAliasInfo, keep_input_mutations: bool, is_train: bool):
```

Comment:
It seems like we have a few variants of this function. Can we just have a single function `_is_mutation_recorded_in_the_graph()` that takes in all the necessary args and is the source of truth for whether a given mutation will be in the graph vs. the runtime epilogue?